docs/3. Implementation of scoring models.md

```diff
@@ -1,14 +1,27 @@
 # Implementation of scoring models #
-Scoring Engine supports scoring models in 3 formats: Java, PMML or R code. Added codes can be queried by engine or used in scenarios.
+Scoring Engine supports scoring models in 4 formats: Java, PMML, Python or R code. Added codes can be queried by engine or used in scenarios.
 
-### Adding new code ###
-Scoring code can be added in Scoring code management panel. To add a scoring code, click the Add New Scoring Code button, then enter the name of the code and drag the file (or a .zip file in case of R models) with the code to designated area on the right side of the screen, or click this area and select the file with the code to add. A .zip file containing R model must contain:
+### Upload new model ###
+Scoring code can be added in Scoring code management panel. To add a scoring code, click the Scoring code action button and select Upload new model option, then enter the name of the code and drag the file (or a .zip file in case of R models) with the code to designated area on the right side of the screen, or click this area and select the file with the code to add. 
+
+A .zip file containing R model must contain:
```

Comment:
Model pythonowy jest dosyć podobny do R w takim sensie, że też się uploaduje zipa z określonymi plikami. Też by się przydało to opisać. Podeślę ci jakiś prosty przykład i wytłumaczę jak się takie modele przygotowuje